daml-foundations/daml-ghc/src/DA/Daml/GHC/Damldoc/Render/Markdown.hs

```diff
@@ -33,12 +33,14 @@ renderSimpleMD ModuleDoc{..} = T.unlines $
   , if null md_classes
     then []
     else [ "## Typeclasses"
+         , ""
+         , "Typeclasses in DAML are an interface that defines some behavior."
```

Comment:
Isn't this mixture of singular and plural odd?

Reply:
yeah, you're right